src/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -70,6 +70,8 @@ fuzz.options.label.threads  = Concurrent scanning threads per scan:
 fuzz.options.title          = Fuzzer
 
 fuzz.category.custom        = Custom fuzzers
+fuzz.options.button.export  = Export
+fuzz.options.button.export.message = Export successful!
```

Comment:
Since these messages apply only to HTTP Fuzzer it's better to add them in the same place as the other ones, for example, along with "fuzz.httpfuzzer.results.toolbar.*".